src/json.ts

```diff
@@ -0,0 +1,33 @@
+import fs, { PathLike } from 'fs';
+
+export const SPACES_TO_INDENT_JSON = 2;
+
+export function parseJsonFile(jsonFile: PathLike, allowMissing: boolean = true): object {
```

Comment:
eslint doesn't seem to like this: weird.

Still, LGTM.

Reply:
It's because the boolean ia already inferred from the = true, so unless the type could be `boolean|object` for instance, it's saying you may aswell drop it.

Reply:
If this is normal in typescript (default parameter value restricting parameter type), then no problem :+1:

Reply:
I disabled this in the `eslintrc` because heck, I do not want them to "infer" it here. Being more strict is always better IMHO and thus I disabled this at all.